### ISSUE TITLE
Sound Times Reporting Incorrectly

### ISSUE THREAD

**Issue by maryfries** [jmoenig/Snap#2461]
We are trying to work a sound project into BJC, and we are running into issues with the way sound lengths are being reported. First, I saw this in a project that Brian sent me as an XML (the first sound came with the file from him, and the second I had recorded after opening it):

![image](https://user-images.githubusercontent.com/6549787/61595977-b252ea80-abcb-11e9-8427-47c7d62971c2.png)

Then I exported the project and imported it again, and both sounds, though they still played, were listed as 0:00:

![image](https://user-images.githubusercontent.com/6549787/61596007-e9c19700-abcb-11e9-8581-f92e0483516e.png)

**Comment by jmoenig** [comment]
yep, that's a somewhat longstanding issue. I changed the title to make it clear that this does, in fact, not affect the new *reporter*, which to my knowledge reports the sound duration correctly. Thanks
